extensions/smallrye-openapi/runtime/pom.xml

```diff
@@ -40,6 +40,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-swagger-ui</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
```

Comment:
Why was adding this dep nessesary?

Reply:
there's a dependency between deployments already, without it the TCK failed with missing `SecurityIdentityAssociation` class. That's probably because of the dependency between RestEasy and security extensions. I check if it's still needed after fixing all the issues in the PR

Reply:
still needed at this point.

Reply:
Can you re-check after making the dependency changes I mention above?